lib/src/core/app_initializer.dart

```diff
@@ -1,7 +1,17 @@
+import 'dart:developer';
+
+import 'package:sequence/src/datasource/local/database.dart';
+
 class AppInitializer {
   Future<void> preAppRun() async {}
 
   Future<void> run() async {}
 
-  Future<void> postAppRun() async {}
+  Future<void> postAppRun() async {
+    final db = AppDatabase();
+
+    final allRecognitions = await db.select(db.recognitionResultDb).get();
```

Comment:
I guess this is to cache our recognitions.
I Will recommend to use `.then` instead of `async`.

**async**
Will interrupt the execution flow until the future is resolved

**then**
Will execute the next steps and trigger a callback when the future is resolved. Which is somehow what we expect in our case. We don't want our app to lag while we are loading  these data.

So here is my suggestion:
```suggestion
    late final allRecognitions;
    db.select(db.recognitionResultDb).get().then((result) => allRecognitions = value);
```